docs/tutorials/run-kinto.rst

```diff
@@ -0,0 +1,107 @@
+.. _run-kinto:
+
+Run Kinto
+#########
+
+Mozilla demo server
+===================
+
+A Kinto instance is running at https://kinto.dev.mozaws.net/v1/
+
+It should be enough to get started, but the records are flushed every night.
+
+
+Using Docker
+============
+
+If you have Docker, Kinto can be started locally with a single command:
+
+::
+
+    sudo docker run -p 8888:8888 kinto/kinto-server
+
+The server should now be running on http://localhost:8888
+
+It is possible to specify every Kinto setting through environment variables.
+For example, using an environment file:
+
+::
+
+    # kinto.env
+    KINTO_USERID_HMAC_SECRET = tr0ub4d@ur
+    KINTO_BATCH_MAX_REQUESTS = 200
+
+And running the container with ``docker run --env-file ./kinto.env ...``
+
+
+Using Python package
+====================
+
+Python tooling
+--------------
+
+The following tools are necessary to initiate the local installation and use
+our helpers:
+
+* `python-virtualenv`
+
+
+Quick start
+-----------
+
+By default, for convenience, *Kinto* persists the records, permissions and
+internal cache in a **volatile** memory backend. On every restart, the server
+will loose its data, and multiple processes are not handled properly.
+
+But again, it should be enough to get started!
+
+
+Create a python isolated environment (*optional*):
```

Comment:
I'd actually reassert the dependency to `python-virtualenv` with an external link to its installation procedure.